libs/vm-modules/src/math/tensor/tensor.cpp

```diff
@@ -83,9 +95,19 @@ void VMTensor::Bind(Module &module, bool const enable_experimental)
            COMPUTE_CHARGE_COST;
   };
 
+  auto tensor_string_constructor_charge_estimate = [](Ptr<String> const &str) -> ChargeAmount {
+    DataType size = static_cast<DataType>(str->string().size());
```

Comment:
auto const - becase you use static_cast, and it is a constant :)

Reply:
@MissingNO57 I probably missed something obvious, but why does this charge estimator use size and the previous constructor charge estimate use padded_size?

Reply:
This uses Size of string, but previous one was using size of Tensor (Number of elements)